shopfloor_reception/services/reception.py

```diff
@@ -212,8 +226,15 @@ def _select_document_from_lot(self, lot):
             move_lines, self.msg_store.no_transfer_for_lot
         )
 
-    def _select_line(self, picking, line, move, increase_qty_done_by=1):
-        product = line.product_id
+    def _scan_line__find_or_create_line(self, picking, move, qty_done=1):
+        line = fields.first(
+            move.move_line_ids.filtered(
+                lambda l: (
+                    not l.result_package_id
+                    and l.shopfloor_user_id.id in [False, self.env.uid]
```

Comment:
I thought we had to ban the use of this deprecated `shopfloor_user_id` field?
cc @jbaudoux

Reply:
This is the only scenario where we really need it :)